app/static/tests/unit/components/fit/fitTab.test.ts

```diff
@@ -33,6 +33,12 @@ describe("Fit Tab", () => {
                         compileRequired
                     }
                 },
+                graphSettings: {
```

Comment:
to save pain later, want to put this bit of boilerplate into the mocks file? That way when we add more settings we only have to update it once